src/extensions/common/Splitter.tsx

```diff
@@ -17,29 +18,34 @@ const defaultGripInnerThickness = 4
 const defaultGripMargin = 2
 
 const Splitter: FunctionComponent<Props> = (props) => {
-    const {width, height, initialPosition, onChange, adjustable=true} = props
+    const {width, height, initialPosition, onChange, adjustable=true, positionFromRight=false} = props
 
     const [gripPosition, setGripPosition] = useState<number>(initialPosition)
 
     if (!props.children) throw Error('Unexpected: no props.children')
 
+    let child1: ReactElement
+    let child2: ReactElement | null
     if (!Array.isArray(props.children)) {
-        let child0: ReactElement = props.children as any as ReactElement
-        return <child0.type {...child0.props} width={width} height={height} />
+        child1 = props.children as any as ReactElement
```

Comment:
Assumption is `props.children`, if it isn't an array, will be a single value.

Do we need to check for it being a non-array object or undefined?

Reply:
Apparently there are a bunch of possibilities. What I did isn't a great way to do this. Could you submit an issue for figuring out how to handle this in a proper way? Right now I am relying on knowledge of what is being sent in as children.

@jsoules